etc/profile-m-z/syncthing.profile

```diff
@@ -0,0 +1,116 @@
+# Firejail profile for syncthing
+# Description: File synchronization using public networks
+# This file is overwritten after every install/update
+
+##quiet
```

Comment:
```suggestion
```

Remove `##` lines and other noise like in #6549.

Same for the rest of the file.